packages/client-sdk-nodejs/src/config/transport/storage/grpc-configuration.ts

```diff
@@ -0,0 +1,64 @@
+export interface StorageGrpcConfigurationProps {
+  /**
+   * The number of internal clients a storage client will create to communicate with Momento. More of them allows
+   * more concurrent requests, at the cost of more open connections and the latency of setting up each client.
+   */
+  numClients?: number;
```

Comment:
unless you are wiring this up in this PR (which is not necessary for mvp) we should remove this for now